app/screens/convert_gm_to_channel/team_selector_list/team_selector_list.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo, useState} from 'react';
+import {View} from 'react-native';
+
+import TeamList from '@app/components/team_list';
+import {useTheme} from '@app/context/theme';
+import {popTopScreen} from '@app/screens/navigation';
+import {changeOpacity, getKeyboardAppearanceFromTheme, makeStyleSheetFromTheme} from '@app/utils/theme';
+import SearchBar from '@components/search';
+
+const getStyleSheet = makeStyleSheetFromTheme(() => ({
+    container: {
+        padding: 12,
+    },
+    listContainer: {
+        marginTop: 12,
+    },
+}));
+
+type Props = {
+    teams: Team[];
+    selectTeam: (teamId: string) => void;
+}
+
+const TeamSelectorList = ({teams, selectTeam}: Props) => {
+    const theme = useTheme();
+    const styles = getStyleSheet(theme);
```

Comment:
with the above suggestion this is not needed, and btw theme is not an argument the function expects.

Reply:
Done.